### PR TITLE
Jaydel/implement movie crud

### PR DESCRIPTION
**Pull request by jidbujawe**
### This PR consist of the following changes:

- Added admin functionalities to add/edit/delete movies
- Working create/edit/delete movie functionalities
- Added appropriate form validations
- Use of Alpine JS to dynamically add casts and character names
- Added actual movie average ratings
- Added movie trailer modal
- Passed all testing

### CODE REVIEW

File: app/Http/Controllers/MovieController.php [hunk 1]
```diff
@@ -19,41 +25,86 @@ public function index()
...
+        return view('movies.create', [
+            'genres' => $genres,
+            'celebs' => $celebs
+        ]);
     }
 
     public function store(MovieRequest $request)
     {
-        Movie::create($request->validated());
+        $movie = Movie::create($request->except('genres', 'celebs', 'characters'));
```
Comment (vrodriguero) [inline app/Http/Controllers/MovieController.php]: Hey Jayd, so after some discussion with Jono and seeing some examples online, the suggestions were to not use expect(). 

So it is best to do like 
```
Movie::create([
'title' => $request->input('title'),
....
]);

File: app/Http/Controllers/MovieController.php [hunk 2]
```diff
@@ -19,41 +25,86 @@ public function index()
...
+            'movie' => $movie,
+            'genres' => $genres,
+            'celebs' => $celebs
+        ]);
     }
 
     public function update(MovieRequest $request, Movie $movie)
     {
-        $movie->update($request->validated());
+        $movie->update($request->except('genres', 'celebs', 'characters'));
```
Comment (vrodriguero) [inline app/Http/Controllers/MovieController.php]: Same comment as the create one Jayd ;D

File: routes/web.php [hunk 3]
```diff
@@ -1,5 +1,6 @@
 <?php
 
+
```
Comment (vrodriguero) [inline routes/web.php]: you can remove this line ;D

File: routes/web.php [hunk 4]
```diff
@@ -20,6 +21,7 @@
 /** Auth */
 Auth::routes();
 
+
```
Comment (vrodriguero) [inline routes/web.php]: this one as well

**Review by vrodriguero — CHANGES_REQUESTED** [review]
Hey Jayd, I have pushed some changes to undo some of the package-lock.json changes. 

I left just a couple of comments :D

File: app/Http/Controllers/MovieController.php (hunk 1 above)
Comment (jbujawe-cybercert) [reply app/Http/Controllers/MovieController.php]: Hey @vrodriguero, I've edited the controller to what you suggested :)

**Review by vrodriguero — APPROVED** [review]
🥳

**Pull request merged**